src/Bridges/Objective/functionize.jl

```diff
@@ -0,0 +1,43 @@
+"""
+    FunctionizeBridge{T}
+
+The `FunctionizeBridge` converts a `SingleVariable` objective into a
+`ScalarAffineFunction{T}` objective.
+"""
+struct FunctionizeBridge{T} <: AbstractBridge end
+function bridge_objective(::Type{FunctionizeBridge{T}}, model::MOI.ModelLike,
+                           func::MOI.SingleVariable) where T
+    F = MOI.ScalarAffineFunction{T}
+    MOI.set(model, MOI.ObjectiveFunction{F}(), convert(F, func))
+    return FunctionizeBridge{T}()
+end
+
+function supports_objective_function(
+    ::Type{<:FunctionizeBridge}, ::Type{MOI.SingleVariable})
+    return true
+end
+MOIB.added_constrained_variable_types(::Type{<:FunctionizeBridge}) = Tuple{DataType}[]
+function MOIB.added_constraint_types(::Type{<:FunctionizeBridge})
+    return Tuple{DataType, DataType}[]
+end
+function MOIB.set_objective_function_type(::Type{FunctionizeBridge{T}}) where T
+    return MOI.ScalarAffineFunction{T}
+end
+
+function MOI.delete(model::MOI.ModelLike, bridge::FunctionizeBridge) end
```

Comment:
What does it mean to delete an objective bridge?

Reply:
I am adding a docstring in https://github.com/JuliaOpt/MathOptInterface.jl/pull/867/

Reply:
See the comment for `ObjectiveSense`. Either add a descriptive comment (`No variables to delete`), or call it `_delete_variables_and_constraints_from_bridge`

Reply:
`MOI.delete` is consistent with the `MOI.delete` of variable and constraint bridges: "you are going to be deleted so first undo anything you have done." so I have kept the name `MOI.delete` and added the comment.